cea/interfaces/dashboard/settings.py

```
@@ -4,11 +4,14 @@
 from pydantic import model_validator, Field
 from pydantic_settings import BaseSettings, SettingsConfigDict
 
+ENV_VAR_PREFIX = "CEA_"
+
+from cea.interfaces.dashboard.lib.cache.settings import CacheSettings
```

Ruff (0.8.2)

Module level import not at top of file

(E402)